docs-internal/Labels.md

```diff
@@ -0,0 +1,56 @@
+# Labels
```

Comment:
❤️ This is really nice! I should add something similar for aragon/aragon 😄

Reply:
Perhaps we can have something like `aragon/community-standards` where anything non-project specific will go there? Like this, `CONTRIBUTING.md`, eslint configs, etc.

Reply:
Yeah, we've been thinking of exactly something like this. Kind of like a "Aragon github standards" that has templates for READMEs, release notes, etc.